wagtail/test/utils/page_tests.py

```diff
@@ -1,18 +1,32 @@
+from typing import Any, Dict, Optional
+from unittest import mock
+
+from django.conf import settings
+from django.contrib.auth.base_user import AbstractBaseUser
+from django.http import Http404
 from django.test import TestCase
 from django.urls import reverse
+from django.utils.http import urlencode
 from django.utils.text import slugify
+from html_form_to_dict import html_form_to_dict
```

Comment:
Considering the test utils are packaged with the default Wagtail, but the requirement is in testing extras, one would need to install `wagtail[testing]` to get the html_form_to_dict dependency.

And we are trying to reduce the regular package dependencies as much as we can

Reply:
Ah yes, you're right. I suppose I could recreate that function in wagtail, but that would still require `lxml` (which could be added to the main dependency list). What do you think @zerolab?

Reply:
@zerolab third-party dep has now been replaced by `wagtail.test.utils.form_data.querydict_from_html()` (bs4-powered equivalent)

Reply:
👍